Modules/System/Time Zone Management/Permissions/TimeZoneMgtObjects.PermissionSet.al

```diff
@@ -0,0 +1,10 @@
+permissionset 132979 TimeZoneMgtObjects
```

Comment:
This PermissionSet should probably use the Id of the normal app `8720`.
The ID Range: 
```json
{
      "from": 132979,
      "to": 132979
    }
```
is only meant for testapps